src/screens/ui/AboutAppletScreen.tsx

```diff
@@ -1,26 +1,45 @@
-import { FC } from 'react';
+import { FC, useMemo } from 'react';
 
+import { useRoute } from '@react-navigation/native';
+
+import { useAppletDetailsQuery } from '@app/entities/applet';
 import rules from '@shared/lib/markDownRules';
 import { MarkdownView } from '@shared/ui';
 import { YStack, ScrollView, Box } from '@shared/ui';
 
+type RouteProps = {
+  key: string;
+  name: string;
+  params: {
+    appletId: string;
+  };
+};
```

Comment:
The types for screens should be declared in `screens/lib/types.ts`
Please take a look at `InProgressActivityScreen.tsx`. There you can find an example of how to connect type the right way.